text/dependencies/rfcs/0002-dependency-server-purl.md

```diff
@@ -0,0 +1,48 @@
+# Add pURL Field to the Dependency Server Metadata
+
+## Summary
+
+Add a field for pURLs into the dependency server schema.
+This RFC is an addendum to [RFC #0009](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0009-dep-server.md)
+
+## Motivation
+
+Currently, the dependency server offers a lot of valuable information about the
+files it hosts. However, it currently does not have the capacity to supply
+pURLs. This field should be added to the dependency server because pURLs are an
+industry standard package identifier that allow dependencies to be accurately
+identified by the end user.
+
+## Implementation
+
+The following field should be added to the dependency server output:
+
+* `purl`: a string which is a pURL for that dependency that conforms to the
+  [pURL Specification](https://github.com/package-url/purl-spec).
+
+An example of the new output:
+```
+$ curl -sL https://api.deps.paketo.io/v1/dependency?name=go
+[
+  {
+    "name": "go",
+    "version": "1.15",
+    "sha256": "29d4ae84b0cb970442becfe70ee76ce9df67341d15da81b370690fac18111e63",
+    "uri": "https://deps.paketo.io/go/go_1.15_linux_x64_bionic_29d4ae84.tgz",
+    "stacks": [
+      {
+        "id": "io.buildpacks.stacks.bionic"
+      },
+      {
+        "id": "io.paketo.stacks.tiny",
+        "mixins": ["some-required-mixin"]
+      }
+    ],
+    "source": "https://dl.google.com/go/go1.15.src.tar.gz",
+    "source_sha256": "69438f7ed4f532154ffaf878f3dfd83747e7a00b70b3556eddabf7aaee28ac3a",
+    "deprecation_date": "",
+    "licenses": ["BSD-3-Clause"],
+    "purl": "pkg:generic/go@1.15?download_url=https://dl.google.com/go/go1.15.src.tar.gz&checksum=sha256:69438f7ed4f532154ffaf878f3dfd83747e7a00b70b3556eddabf7aaee28ac3a"
```

Comment:
This example is good, but it might be helpful to go into more detail in the implementation section that we are specifically going to be using the `generic` type, rather than something else (like the `debian` type) and what the other bells and whistles here are (the checksum, the download URL)

This is helpful so we have a standard we can apply to all of our dependencies consistently.

Reply:
I have just pushed up a change. Give it a read and see if it satisfies this comment.